### PR TITLE
Create AKS Kubeconfig secret

### LINKED ISSUE #8254: Create Kubeconfig Secret for AKS cluster
**User Story**
Fetch Kubeconfig for AKS cluster and save it in a Kubernetes Secret

**Acceptance criteria**

### PR DESCRIPTION
**Pull request by imharshita**
Signed-off-by: Harshita <imharshita@github.com>

**What this PR does / why we need it**:
Create AKS Kubeconfig secret

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #8254

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: pkg/resources/credentials.go [hunk 1]
```diff
@@ -295,6 +295,60 @@ func GetAWSCredentials(data CredentialsData) (AWSCredentials, error) {
 	return awsCredentials, nil
 }
 
+func GetAWSEKSCredentials(ctx context.Context, client ctrlruntimeclient.Client, cluster *kubermaticv1.ExternalCluster) (AWSCredentials, error) {
```
Comment (zreigz) [inline pkg/resources/credentials.go:313]: I like those methods. Can you also move GKE here to have consistency

Comment (imharshita) [reply pkg/resources/credentials.go:313]: sure

Comment (zreigz) [reply pkg/resources/credentials.go:313]: thx

Comment (imharshita) [reply pkg/resources/credentials.go:313]: done

Comment (zreigz) [reply pkg/resources/credentials.go:313]: waiting for tests

**Pull request merged**